### ISSUE TITLE
v2 is built with Java 8

### ISSUE THREAD

**Issue by kirillzh** [anshooarora/extentreports-java#678]
Even though documentation states that Extent v2 supports Java 7, it seems to be built using Java 8.
I'm compiling from Gradle: `compile 'com.relevantcodes:extentreports:2.41.2'`

**Comment by kirillzh** [comment]
Seems to be this commit? https://github.com/anshooarora/extentreports-java/commit/cdf2d2c948a90a916caeaaa258695e3442e9a819

**Comment by anshooarora** [comment]
This is correct.  Its a defect.  Its supported to be built with JDK6.  Will release a fix very soon.

**Comment by haroon-sheikh** [comment]
@anshooarora would you be able to release this fix for us?

**Comment by anshooarora** [comment]
@haroon-sheikh  -  Merged your commit.  I will release the new version soon.

**Comment by BhavithaSundharesan** [comment]
Hi @anshooarora ,

Is there the new version?

**Comment by BhavithaSundharesan** [comment]
Hi @anshooarora 
It s that our project have to use java 7 and extent 3 needs java 8
I have downloaded jars for 2.41.2 but it still throws error as 'com/relevantcodes/extentreports/ExtentReports : Unsupported major.minor version 52.0
'

**Comment by slrautomation2** [comment]
Hi Anshoo,
Which java version should be used with 2.41.2.
I am using 2.41.2 with python, java 8 and getting below error.

java.lang.RuntimeException: Class com.relevantcodes.extentreports.ExtentReports not found

**Comment by haroon-sheikh** [comment]
@BhavithaSundharesan Please use `2.41.3` instead
